config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -154,6 +154,7 @@ periodics:
     preset-service-account: "true"
     preset-common-gce-windows: "true"
     preset-e2e-gce-windows: "true"
+    preset-windows-repo-list-2004: "true"
```

Comment:
Where is this preset defined? I didn't see anything in [test-infra](https://github.com/kubernetes/test-infra/search?q=windows-repo-list&type=code) nor [windows-testing](https://github.com/kubernetes-sigs/windows-testing/search?q=windows-repo-list&type=code).

Reply:
Oh, looks like it's [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml#L21) - github code search just sucks.